Extension/src/LanguageServer/client.ts

```diff
@@ -2660,6 +2661,25 @@ export class DefaultClient implements Client {
         }
     }
 
+    public async handleCheckForCompiler(): Promise<void> {
+        await this.awaitUntilLanguageClientReady();
+        const compilers: configs.KnownCompiler[] | undefined = await this.getKnownCompilers();
```

Comment:
@Colengms These compilers don't have the correct slash direction -- can you fix that via something like what is done in the setKnownCompilers method?